content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Agent Configuration
+weight: 45
+spelling: cSpell:ignore distro mkdir uninstrumented virtualenv
+---
+
+Automatic instrumentation with Python uses a Python agent binary that can be attached to any Python application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks.
+
+## Setup
```

Comment:
Imho, "setup" belongs into the _index.md

Reply:
@svrnm @mviitane I rejigged things a bit, so that it matches more with how the Java docs are structured. Let me know if this is what y'all had in mind.